src/Basis/Constructors.jl

```diff
@@ -136,7 +136,7 @@ wsum = sum(mweights);
 
 ```
 """
-function transformquadrature(points, weights = nothing, mapping = nothing)
+function transformquadrature(points, weights = nothing, mapping::function)
```

Comment:
You just made it a required argument instead of an optional argument. You need
```julia
function transformquadrature(points, weights = nothing, mapping::Function = nothing)
```

Reply:
ok, alright! Pls, what about ```Function``` or ```function```
Should it be ```F``` or ```f```?

Reply:
How could you find out in 10 seconds at a Julia terminal?